assets/_locales/en/messages.json

```diff
@@ -4390,6 +4390,10 @@
     "message": "Swap error. Please try again.",
     "description": "Generic error message shown when a swap transaction fails" 
   },
+  "swap_error_increase_slippage": {
```

Comment:
Maybe improve error messaging ? It's used for permaswap which is thrown if the expected amountOut is greater than the actual amount Out from permaswap.

Reply:
Let me update that. This message is better.